schema/project-establishment.js

```diff
@@ -0,0 +1,37 @@
+const BaseModel = require('./base-model');
+const { uuid } = require('../lib/regex-validation');
+
+const statuses = [
+  'draft',
+  'active',
+  'removed'
+];
+
+class ProjectEstablishment extends BaseModel {
+  static get tableName() {
+    return 'projectEstablishments';
+  }
+
+  static get idColumn() {
+    return ['projectId', 'establishmentId'];
```

Comment:
Is this compatible with soft deleting things?

Reply:
we are hard deleting these

Reply:
Then we probably don't want the `deleted` column on the table.

Reply:
correct!